### PR TITLE
updating version of bwctl

### PR DESCRIPTION
**Pull request by bltierney**
A new version of bwctl was recently released.

### CODE REVIEW

File: Library/Formula/bwctl.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class Bwctl < Formula
   homepage "http://software.internet2.edu/bwctl/"
-  url "http://software.internet2.edu/sources/bwctl/bwctl-1.5.2-10.tar.gz"
+  url "http://software.internet2.edu/sources/bwctl/bwctl-1.5.4.tar.gz"
   sha1 "5dcc7a1d671ac8e061f859a430d56ae2551f507e"
```
Comment (mikemcquaid) [inline Library/Formula/bwctl.rb]: This `sha1` will need updating (and ideally changed to a `sha256`)

**Comment by bltierney** [comment]
ready to pull now

**Comment by DomT4** [comment]
Looks like it's installing a local ` i2util `, which is attempting to override Homebrew's.

```
configure: WARNING: Using local I2util
==> Finishing up
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/aespasswd
Target /usr/local/bin/aespasswd
is a symlink belonging to i2util. You can unlink it:
  brew unlink i2util

To force the link and overwrite all conflicting files:
  brew link --overwrite bwctl
```

**Comment by tdsmith** [comment]
It looks like we'll need to pass `"--with-I2util=#{Formula["i2util"].opt_prefix}"` to configure, and patch configure with:

```
# configure mis-sets CFLAGS for I2util
# https://lists.internet2.edu/sympa/arc/perfsonar-user/2015-04/msg00016.html
inreplace "configure", 'CFLAGS="-I$I2util_dir/include $CFLAGS"', 'CFLAGS="-I$with_I2util/include $CFLAGS"'
```
at the top of the `install` method.

I reported this upstream here: https://lists.internet2.edu/sympa/arc/perfsonar-user/2015-04/msg00016.html

**Pull request closed without merging**